File: cpp/lv1/use_string_basic.cpp
//문자열 다루기 기본

#include <string>
#include <vector>

using namespace std;

/* Answer 1.
bool solution(string s) {
    
    if(s.length() == 4 || s.length() == 6)
    {
        for(int i = 0; i < s.length(); i++)
        {
            if(isalpha(s[i]) != 0)
            {
                return false;
            }
        }
     return true;   
    }
    return false;
} */


/* Answer 2.
bool solution(string s) {

for (int i = 0; i < s.size(); i++)
    {
        if (!isdigit(s[i]))
            return false;
    }

    return s.size() == 4 || s.size() == 6 ? true : false;
} */

File: python/lv1/use_string_basic.py
# 문자열 다루기 기본

'''
Answer 1.
def solution(s):

    if len(s) == 4 or len(s) == 6:
        for i in range(0, len(s)):
            if s[i].isalpha():
                return False
        return True
    else:
        return False
'''

'''
Answer 2.
def solution(s):
    return s.isdigit() and len(s) in [4, 6]
'''
    
'''
Answer 3.
def solution(s):
  try:
      int(s)
  except:
      return False
  return len(s) == 4 or len(s) == 6
'''
  
